cmd/tidb-server/fips.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build boringcrypto
+// +build boringcrypto
+
+package main
+
+import _ "crypto/tls/fipsonly"
+
+import "github.com/pingcap/tidb/pkg/parser/mysql"
+
+func init() {
+	mysql.TiDBReleaseVersion += "-fips"
```

Comment:
where is this variable used?

Reply:
When output TiDB information, for example, use './bin/tidb-server -V '

Reply:
./bin/tidb-server -V
Release Version: v7.6.0-alpha-4-g31ea0f1009-dirty-fips
...
@CabinfeverB

Reply:
https://github.com/tiancaiamao/tidb/blob/server-fips/cmd/tidb-server/main.go#L877C18-L877C18
Used by './bin/tidb-server -V' to print the version information @coderplay